### PR TITLE
feat: adds settings page and fixes tests output

### PR DESCRIPTION
**Pull request by jaysoo**
- Test output will capture build errors now.
- New setting for enabling/disabling detailed status (defaults to true).

### CODE REVIEW

File: libs/feature-settings/src/lib/feature-settings.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```
Comment (mrmeku) [inline libs/feature-settings/src/lib/feature-settings.component.spec.ts]: Remove this empty spec file or add some tests

File: libs/feature-settings/src/lib/feature-settings.component.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'angular-console-settings-feature-settings',
+  template: `
+    <p>
+      feature-settings works!
```
Comment (mrmeku) [inline libs/feature-settings/src/lib/feature-settings.component.ts]: This looks not very functional?

File: server/src/schema/resolvers.ts [hunk 3]
```diff
@@ -204,14 +204,18 @@ const Database: DatabaseResolvers.Resolvers = {
   },
   commands(_root: any, args: any) {
     try {
+      const settings = readSettings();
+      const includeDetailedStatus = settings.enableDetailedStatus;
       if (args.id) {
-        const c = commands.recent.find(c => c.id === args.id);
+        const c = commands.recent.find((c: any) => c.id === args.id);
```
Comment (mrmeku) [inline server/src/schema/resolvers.ts]: Do we have an interface we can use in place of any?

File: libs/feature-settings/src/lib/feature-settings.component.ts (hunk 2 above)
Comment (jaysoo) [reply libs/feature-settings/src/lib/feature-settings.component.ts]: Will remove!

File: libs/feature-settings/src/lib/feature-settings.component.spec.ts (hunk 1 above)
Comment (jaysoo) [reply libs/feature-settings/src/lib/feature-settings.component.spec.ts]: Good point

**Pull request merged**